src/BlazorDB/IndexDbManager.cs

```diff
@@ -300,7 +300,43 @@ public async Task<IList<TRecord>> ToArray<TRecord>(string storeName)
 
             return default;
         }
+
+        /// <summary>
+        /// Filter a store on an indexed value 
+        /// </summary>
+        /// <param name="storeName">The name of the store to retrieve the records from</param>
+        /// <param name="indexName">index field name to filter on</param>
+        /// <param name="filterValue">filter's value</param>
+        /// <returns></returns>
+        public async Task<IList<TRecord>> Where<TRecord>(string storeName, string indexName, object filterValue)
+        {
+            var filters = new List<IndexFilterValue>() {new(indexName, filterValue)};
```

Comment:
What version of C# is this?  I don't think I'm using it yet, but I'm always happy to update this.

Reply:
It's a C# 9.0 feature, maybe not ideal to use on a project that target .Net Core 3.1 but it is compatible with.